tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -97,32 +104,31 @@ def test_record_result_requires_dict_result(self, svc, hook_result):
                 self.GRADING_ID, pre_record_hook=lambda *args, **kwargs: hook_result
             )
 
-    def test_it_signs_request_with_oauth1(self, svc, requests, oauth1_service):
-        requests.post.side_effect = OSError()
+    def test_it_signs_request_with_oauth1(self, svc, http_service, oauth1_service):
+        http_service.request.side_effect = OSError()
 
         # We don't care if this actually does anything afterwards, so just
         # fail here so we can see how we were called
         with pytest.raises(OSError):
             svc.record_result(self.GRADING_ID)
 
-        requests.post.assert_called_with(
+        http_service.request.assert_called_once_with(
+            "POST",
             url=Any(),
             data=Any(),
             headers=Any(),
             auth=oauth1_service.get_client.return_value,
-            timeout=Any(),
         )
 
-    def test_requests_fail_if_http_status_is_error(self, svc, respond_with):
-        respond_with(status=400)
+    def test_requests_fail_if_the_third_party_request_fails(self, svc, http_service):
+        http_service.request.side_effect = HTTPError
```

Comment:
`test_requests_fail_if_http_status_is_error()` was removed because checking the HTTP status code and raising if it's an error (4xx or 5xx) is no longer `LTIOutcomesService`'s job. `HTTPService` handles that now. `LTIOutcomesService` just needs to deal with `HTTPService` raising `HTTPError`. I added this new test for that